### DISCUSSION TITLE
WLED as OpenKNX?

### DISCUSSION THREAD

**Discussion by Alex0176** [OpenKNX/OpenKNX#12] [Ideas]
Hallo, 
did anyone build a hardware for WLED and KNX? 
There is a project as user addon:
https://github.com/PaoloTK/WLED/tree/KNX/usermods/KNX

It will be cool to power the ESP32 from KNX and a relay to turn on/off the power supply for the LED Strips or connect it via group addresses to a actor. 

best regards

Alex

**Comment by cornelius-koepp** [comment]
@Alex0176 we now have a OpenKNX module ([OFM-NeoPixel](https://github.com/OpenKNX/OFM-NeoPixel)) and OpenKNX application ([OAM-NeoPixel](https://github.com/OpenKNX/OAM-NeoPixel)) for addressable LED-stripes, but still in development-state.
